scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -23,19 +23,80 @@ source "$(dirname "${0}")"/../functions
 L_PWD=$(pwd)
 cd "${DEVON_IDE_HOME}" || exit 255
 
+# Install Eclipse plugins by feature group id
 # $1: plugin feature group ID
 # $2: plugin URL (update site)
 function doInstallEclipsePlugin() {
-  local plugin="${1/\.feature\.group*/}"
-  echo "Installing eclipse plugin ${plugin} from ${2}"
-  # shellcheck disable=SC2086
-  "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -repository "${2}" -installIU "${1}" -vmargs ${ECLIPSE_VMARGS}
-  result=${?}
-  if [ "${result}" != 0 ]
+  # Remove feature.group from each plugin id
+  local plugins="${1//.feature.group/}"
+  echo "Installing Eclipse plugin '${plugin_name}' from URL ${2}"
```

Comment:
after reviewing this I wonder what happened here (even before your change and what I had done myself here long time ago)...

`plugin_name` is not defined within this function but in `doAddPlugin` or `doAddPlugins` as a side-effect. This is not very clean or intuitive to understand. Maybe it would be better to pass it as an explicit argument and always declare it as local variable...

However, your change is good to print the simpler name for betting understanding. However, in case someone now calls `devon eclipse add-plugin net.sf.eclipsecs.feature.group https://checkstyle.org/eclipse-cs-update-site` this used to print out `net.sf.eclipsecs` and with your change will print `net.sf.eclipsecs.feature.group`. So if we would pass `plugin_name` as 3rd param (`$3`)  and if undefined fallback to `"${1/\.feature\.group*/}"` we would get the best result and maintainability.

Reply:
Thank you for pointing out an really important issue, that I haven't considered yet. I have removed the `plugins="${1/\.feature\.group*/}"` line from the `doInstallEclipsePlugin()` function altogether, because only `doCheckPluginsInstalled()` needs to strip the strings down to their base name.